webpack.config.babel.js

```diff
@@ -98,6 +97,8 @@ const PROD_CONFIG = {
   'BUNDLES_PATH' : `https://s3-eu-west-1.amazonaws.com/onfido-assets-production/web-sdk-releases/${packageJson.version}/`,
```

Comment:
As @rfreitas suggested yesterday, it will make more sense to have `BUNDLES_PATH` renamed as `PUBLIC_PATH`